internal/api/v1/organizations.go

```diff
@@ -225,7 +217,8 @@ func (oc NamespacesController) Delete(w http.ResponseWriter, r *http.Request) AP
 
 	w.Header().Set("Content-Type", "application/json")
```

Comment:
same

Reply:
Yes, again bypassing the SetHeader in jsonResponse, because I didn't want to make this PR even bigger.